src/test/java/com/bouncestorage/swiftproxy/v1/ObjectResourceTest.java

```diff
@@ -76,6 +76,24 @@ public void testPut() throws Exception {
     }
 
     @Test
+    public void testMissingContainer() throws Exception {
+        Response resp = target.path(Joiner.on("/").join(TestUtils.ACCOUNT_PATH, "foo", "bar"))
+                .request()
+                .header("x-auth-token", authToken)
+                .get();
+        assertThat(resp.getStatus()).isEqualTo(Response.Status.NOT_FOUND.getStatusCode());
+    }
+
+    @Test
+    public void testMissingContainerHEAD() throws Exception {
+        Response resp = target.path(Joiner.on("/").join(TestUtils.ACCOUNT_PATH, "foo", "bar"))
+                .request()
+                .header("x-auth-token", authToken)
+                .head();
+        assertThat(resp.getStatus()).isEqualTo(Response.Status.NOT_FOUND.getStatusCode());
+    }
+
+    @Test
```

Comment:
Swift tests do have a test for this. Looks like this only comes up with jclouds transient and file system blob stores (https://issues.apache.org/jira/browse/JCLOUDS-193). I will remove the test from the PR.